packages/rest/tests/did.test.ts

```diff
@@ -0,0 +1,50 @@
+import type { Agent, DidResolutionResult } from '@aries-framework/core'
+import type { Server } from 'net'
+
+import request from 'supertest'
+
+import { startServer } from '../src'
+
+import { getTestAgent, getTestDidRecord, objectToJson } from './utils/helpers'
+
+describe('DidController', () => {
+  let app: Server
+  let aliceAgent: Agent
+  let didRecord: DidResolutionResult
+
+  beforeAll(async () => {
+    aliceAgent = await getTestAgent('Connection REST Agent Test Alice', 3999)
+    app = await startServer(aliceAgent, { port: 3000 })
+    didRecord = await objectToJson(getTestDidRecord())
```

Comment:
We don't need to create a did record here. The did resolver dynamically resolves the document, and isn't actually using the storage. We can just use the did document itself

Reply:
Got it